src/test/java/com/softserveinc/ita/mmakoviichuk/SoftServeTest.java

```diff
@@ -18,6 +18,7 @@ public void open() {
     public void sidebarSingleClickTest() {
         SoftServePage softServePage = new SoftServePage();
         softServePage.switchSidebarSection("Careers");
-        Assert.assertTrue(softServePage.isSidebarSwitched());
+
+        Assert.assertTrue(softServePage.isSidebarSwitched("Careers"));
```

Comment:
use assertThat from assertJ lib with such pattern:

assertThat(softServePage.isSidebarSwitched("Careers"))
.as("Careers" + " sidebar should be switched") <- Message if verification will fail for example
.isTrue();